scripts/secure-boot/generate-images/intermediate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# go into a separate directory to make cleanup easier
+mkdir -p something
```

Comment:
@WiktorG351 in this situations we usually create dir named `tmp`. Even better is to run `mktemp -d`

Reply:
@TomaszAIR adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/d5daafb1fbd4dbfc97289aa9eebda2b7e6c8f7a6#diff-00f53487a7d91b72da783fcf7286011bd2c964cb60496c9c6e607a55e9ae748dR3)